spec/flows/user_confirmation_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe UserConfirmationFlow, type: :model do
+  describe "#perform" do
+    it "confirms the user if there is receivers available" do
+      Settings.put(Settings::REGISTRATIONS_ENABLED, true)
```

Comment:
Can we extract this to a `enable_registrations` method? I mean, is it possible? I don't recall very much on RSpec.

Reply:
Yes